walkthroughs/week-5-tdd/project/src/main/java/com/google/sps/FindMeetingQuery.java

```diff
@@ -66,6 +66,42 @@
       availableTimes.add(TimeRange.fromStartEnd(startPointer, TimeRange.END_OF_DAY, true));
```

Comment:
I just noticed that END_OF_DAY is actually 23:59, do you actually want to use WHOLE_DAY for this calculation instead? e.g. is a whole_day event supported?

Reply:
Using WHOLE_DAY there would be applicable only if there were no events that mandatory attendees attend. Lines 64-67 check to see if after the last mandatory attended event there is a gap big enough for an event between startPointer and the end of the day. By the algorithm if outOfRange has at least one event, than startPointer by lines 64-67 would be eventTime.end(), or the end of the last event, then I would not want to use WHOLE_DAY. If outOfRange has no events, then startPointer would be START_OF_DAY, then yes, I would want the calculation to give back WHOLE_DAY.

Reply:
Could you add tests for whole day events or events that end after 23:59?

Reply:
I added three tests,: wholeDayEventBothAttend(), PersonAHasWholeDayEvent() and EventEndsAfterEndOfDay(). The first two tests for whole day events and the third is a test for an event that ends after 23:59.

Reply:
Could you also add test for querying a DURATION_30_MINUTES event when there is an existing event end at TIME_1130PM? Will there be available option?
|--------A--------|         event ends at 23:30
|---------------------|   a day

Reply:
I added the test, because it includes the end of day there will be an available option.